data/phone_formats.json

```
@@ -18,6 +18,7 @@
   "dm": "+1-767-555-1234",
   "do": "+1-809-555-1234",
   "es": "+34 989 12 34 56",
+  "eg": "+20 1 234567890",
```

let's keep the list alphabetically sorted
```suggestion
  "eg": "+20 1 234567890",
  "es": "+34 989 12 34 56",
```